core/src/main/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommitManager.java

```diff
@@ -90,7 +90,10 @@ public TwoPhaseConsensusCommitManager(DatabaseConfig databaseConfig) {
     coordinator = new Coordinator(storage, config);
     parallelExecutor = new ParallelExecutor(config);
     recovery = new RecoveryHandler(storage, coordinator, tableMetadataManager);
-    commit = new CommitHandler(storage, coordinator, tableMetadataManager, parallelExecutor);
+    // Create CommitHandler with throwExceptionIfCommittedTransactionExists false different from the
+    // case for ConsensusCommitManager because each service tries to put the state ‘committed’ in
+    // the two-phase commit transaction, and we want to ignore the existing committed state record.
```

Comment:
I believe we decided that `transactions with a two-phase commit interface` was the accurate phrase. Should this wording match that?

```suggestion
    // the transaction with a two-phase commit interface, and we want to ignore the existing committed state record.
```

Reply:
Yes, exactly speaking. I applied it with formatting in 344aade. Thank you!